docs/guides/repl/Matter - Multi Fabric Commissioning.ipynb

```diff
@@ -41,23 +41,12 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
```

Comment:
.ipynb are somewhat hard to read/convert since they are more 'binary' than text (even though json, the format does not seem meant for humans).

Could we use some sort of conversion like https://nbconvert.readthedocs.io/en/stable/usage.html ?

Reply:
Seems the misspell won't be happy with such change.

If we update the notebook file, it will probably introduce some base64 changes, then misspell will regard it as spelling errors, end up inserting lots of random words into the dictionary.

Since github can render the notebook, maybe we can adopt something like [ReviewNB](https://www.reviewnb.com/#)?